.github/workflows/daily.yml

```
@@ -907,7 +907,7 @@ jobs:
   build-macos:
     strategy:
       matrix:
-        os: [macos-13, macos-15]
+        os: [macos-15, macos-26]
```

is macos-14 available? This CI was originally to test the lowest and highest versions.
```suggestion
        os: [macos-14, macos-26]
```